README.md

```diff
@@ -50,6 +50,10 @@ For everyone else, the detailed instructions to install Alacritty can be found
 Since Alacritty uses OpenGL to render the terminal content on all platforms, it
 is currently not possible to run it without support for at least OpenGL 3.3.
 
+On Windows, a native PTY API was only introduced [in 2018](https://devblogs.microsoft.com/commandline/windows-command-line-introducing-the-windows-pseudo-console-conpty/).
```

Comment:
I don't like linking to a Microsoft blog post here. The amount of actual useful information seems quite limited and it's not actually a proper release announcement of the feature. We'd do better just without.

I also don't think we need any grand explanation or justification. We can just state the dependency on ConPTY and the minimum Windows version required. What came before it or why Microsoft did it is irrelevant. So just stating that Alacritty requires the ConPTY API which comes with XXXX release should be sufficient as a single sentence.

I'd even be curious if it wouldn't be possible to significantly simplify our requirements here. The documentation of OpenGL is already too much for many to read, it should be possible to see at a glance what is necessary to run Alacritty. Maybe something like this:

```
Requirements:
 - OpenGL 3.3 or higher
 - [Windows] ConPTY support (Windows XXXX or higher)
```

I think that makes it easier to read, without even having to actually "read" anything. I'd be curious if @nixpulvis has an opinion on this.